api/swagger/src/definitions/definitions.yaml

```diff
@@ -27,23 +27,19 @@ UpdateEntity:
 Artist:
   properties:
     id:
-      type: integer
-      format: int32
+      type: string
     name:
       type: string
     artist_type:
       type: string
-    picture :
-      type: string
-      format: binary
     dateOfBirth :
       type: string
-      format: date
+      format: yyyy-mm-dd
     placeOfBirth :
         type : string
     dateOfDeath :
       type: string
-      format: date
+      format: yyyy-mm-dd
```

Comment:
Yes , apparently there is a bug in Postgres because of which DATEONLY is handled the same way as DATE is handled.
Consequence : In database we store for Eg:1940-12-20 is the dateOfBirth in DB , but sequelize returns "dateOfBirth":"1940-12-20T23:00:00.000Z" .This causes Swagger to throw a type mismatch error.
What I committed here is not perfect solution as this just avoids the type mismatch error .We still need to make Sequelize return only 1940-12-20. I have found a better solution today .I will make the change .
FYI:https://github.com/sequelize/sequelize/issues/4858